CMakeLists.txt

```diff
@@ -15,11 +15,26 @@ else()
     set(CMAKE_CXX_FLAGS "-Werror -Wno-missing-declarations -O3 -fPIC")
 endif()
 
-#TODO the rest of deps should be moved outside of the project too (if possible)
+# TODO the rest of deps should be moved outside of the project too (if possible)
 MESSAGE("using deps from vcpkg or system")
 find_package(Boost 1.75.0 REQUIRED)
 find_package(OpenSSL REQUIRED)
-find_package(cryptopp CONFIG REQUIRED)
+find_package(CryptoPP REQUIRED)
+
+# TODO remove this logging
+message("CRYPTOPP_LIBRARIES: " ${CRYPTOPP_LIBRARIES})
+
+# we need ${CRYPTOPP_LIBRARIES} variable set, it is not set by find_package + vcpkg
+# in such a case just find it manually
+if(NOT CRYPTOPP_LIBRARIES)
+	# TODO remove this logging
+	message("looking for CRYPTOPP_LIBRARIES...")
+	find_library(CryptoPP_LIBRARY NAMES cryptopp-static cryptopp DOC "CryptoPP library")
+	set(CRYPTOPP_LIBRARIES ${CryptoPP_LIBRARY})
+endif()
+
+# TODO remove this logging
+message("CRYPTOPP_LIBRARIES: " ${CRYPTOPP_LIBRARIES})
```

Comment:
To rm ? :)

Reply:
@TomaszAugustyn will check if everything works correct tomorrow, then please remove.

Reply:
Removed.